docs/monitoring/resources.rst

```diff
@@ -0,0 +1,46 @@
+.. |MonitorDiskUsge| image:: ../images/MonitorDiskUsge.jpg
+
+Resources
+=========
+
+Monitoring resource usage within a Fledge instance can be achieved by using the south plugin that is designed to monitor the Linux resources of the machine on which Fledge is running, this is the systeminfo south plugin.
+
+This plugin creates a number of assets with a configurable prefix on the asset name. This include
+
+  - The aggregate CPU usage for all the CPUs within the machine.
+
+  - The disk traffic for that disk for each disk connected to the machine.
+
```

Comment:
I think this should be "The disk traffic for each disk connected to the machine."